pom.xml

```diff
@@ -181,6 +182,17 @@
       <artifactId>system-stubs-jupiter</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+    <groupId>com.fasterxml.jackson.core</groupId>
+    <artifactId>jackson-databind</artifactId>
+    <version>${jackson.version}</version> <!-- Use the latest version -->
+</dependency>
+<dependency>
+    <groupId>com.fasterxml.jackson.core</groupId>
+    <artifactId>jackson-core</artifactId>
+    <version>${jackson.version}</version>
```

Comment:
This must be test scope right? We are using GSON as JSON serializer: https://github.com/oras-project/oras-java/blob/main/src/main/java/land/oras/utils/JsonUtils.java#L30

I don't want to mix several JSON provider

Reply:
Removed thanks @jonesbusy